interview-questions.md

```diff
@@ -9,21 +9,36 @@ Then, research the question to expand on your answer. Even if you feel you have
 
   Your answer:
 
-  Researched answer:
+You can fix the problem by using a student ID number as primary key in the Student model. You can assign this as the foreign key for the Cohort model. 
 
+  Researched answer:
+The foreign key will usually be the primary keys of another table. If the Cohort model was created after the Student model you can use the primary keys in the Student model as the foreign key to the Cohort model. For example a student ID number may have been used as a primary key for the Student model.
+ To fix you will have to add another column to Cohort model for the student ID number using migration. And then establish the relationship by adding a belongs_to method.
 
 
 2. Which RESTful routes must always be passed params? Why?
 
-  Your answer:
+  Your answer: 
+
```

Comment:
Checkout the restful routes diagram
https://github.com/learn-academy-2022-charlie/Syllabus/blob/main/rails/restful-routes-crud.md